app/views/layouts/application.html.erb

```diff
@@ -42,7 +42,9 @@
           <%= yield :header %>
         </header>
 
-        <%= yield %>
+          <div class="flow-content">
+            <%= yield %>
+          </div>
```

Comment:
the problem here is that this applies to all the content and not just the frontend ones

Reply:
The flow-content has now been dynamically added to pages that don't come from /admin controllers 😊